src/index.ts

```diff
@@ -64,29 +64,31 @@ export function _buildMetadataAccessor(type: string) {
     rax.attach(ax);
     const baseOpts = {
       url: `${BASE_URL}/${type}${property}`,
-      headers: {'Metadata-Flavor': 'Google'},
+      headers: {'metadata-flavor': 'Google'},
```

Comment:
Why is this change being made? The existing case was correct as per [the documentation](https://cloud.google.com/compute/docs/storing-retrieving-metadata).